modules/azure_uc/providers.tf

```diff
@@ -0,0 +1,8 @@
+provider "azurerm" {
```

Comment:
shouldn't have `provider` blocks within a module - https://developer.hashicorp.com/terraform/language/modules/develop/providers

Reply:
@nathanknox , can you comment on this? If we don't provide it here, I think we don't define anything in the file so I can delete it. Sound right?

Reply:
Yes, this should go away. I think I copied it in by accident or was testing at one point. Should have caught it myself, but missed it.

Reply:
I removed it.